JS-108_javascript-basics/code-challenge/05-the-conditional-challenge/js/quiz.js

```diff
@@ -0,0 +1,74 @@
+/* 
+  1. Store correct answers
+   - When quiz begins, no answers are correct
+*/
+let score = 0;
+
+// 2. Store the rank of a player
+const RANK_GOLD = 'Gold';
+const RANK_SILVER = 'Silver';
+const RANK_BRONZE = 'Bronze';
+const RANK_NONE = 'No crown';
+let rank = RANK_NONE;
+
+// 3. Select the <main> HTML element
+const main = document.querySelector('main');
+
+/*
+  4. Ask at least 5 questions
+   - Store each answer in a variable
+   - Keep track of the number of correct answers
+*/
+// A debate from Looney Toons
+const answer1 = prompt('Is the Earth round or flat? (round or flat)');
+
+if ( answer1.toLowerCase() === 'round' ) {
+  score += 1;
+}
+// Put a programming question
+const answer2 = prompt('Boolean data type evaluates to either true or false. (true or false)');
+
+if ( answer2.toLowerCase() === 'true' ) {
+  score += 1;
+}
+// Sing a song
+const answer3 = prompt('Twinkle, twinkle little...');
+
+if ( answer3.toLowerCase() === 'star' ) {
+  score += 1;
+}
+// A - Z
+const answer4 = prompt('Which comes first, the chicken or the egg? (chicken or egg)');
```

Comment:
I'm not sure if there's a correct answer to this. But I can let this pass this time only. 🤣